doc/source/whatsnew/v0.18.1.txt

```diff
@@ -131,6 +131,7 @@ API changes
 
 
 
+- the default for ``.query()/.eval()`` is now ``engine=None`` which will use ``numexpr`` if its installed, else will fallback to the ``python`` engine. This mimics the pre-0.18.1 behavior if ``numexpr`` is installed (and which previously would raise if ``numexpr`` was NOT installed and ``.query()/.eval()`` was used). (:issue:`12749`)
```

Comment:
@jorisvandenbossche @TomAugspurger wording make sense? This is really a bug fix, but wanted to highlite it.